department-of-veterans-affairs/caseflow
# Add MDR to CAVC Remand Entry Form

Resolves #15384 / [CASEFLOW-94](https://vajira.max.gov/browse/CASEFLOW-94)

### Description
Please explain the changes you made here.

### Acceptance Criteria

### Testing Plan
1. Go to (a realistic task tree, dispatched, AMA case)
2. Hit `+ Add CAVC Remand`
3. Fill out the form, **skipping Judge selection, judgement date, and mandate date**
4. Confirm that the infobox appears when MDR is selected
5. Hit submit - confirm Judgement Date and Mandate Date **do not** display a JS validation error requiring input
6. Fill out Judge, Judgement Date, and Mandate Date
   - Judgement Date and Mandate date are still required for the _back end_ validation - will be fixed in [CASEFLOW-97](https://vajira.max.gov/browse/CASEFLOW-97)
7. Hit submit and confirm the success message has the MDR wording, and the CAVC section lists the `Type`: `Memorandum Decision on Remand (MDR)`
   - Again, we are not checking if the MDR flow created the right tasks - that's later in the epic!


### User Facing Changes

**Info box on MDR selection**
![image](https://user-images.githubusercontent.com/6129479/103581015-2dd05680-4ea9-11eb-8c11-13b9c513737b.png)

**New Success detail**
![image](https://user-images.githubusercontent.com/6129479/103580950-1729ff80-4ea9-11eb-84b7-f73f44c1b0dc.png)


### Storybook Story
*For Frontend (Presentationa) Components*
